oc-chef-pedant/spec/api/account/account_acl_spec.rb

```diff
@@ -29,7 +29,7 @@
         it "can get user acl" do
           get(request_url, platform.superuser).should look_like({
              :status => 200,
-             :body_exact => acl_body
+             :body => acl_body
```

Comment:
@jkeiser Can you provide a more detailed commit message that explains why this is correct since it is presumably making the test less strict and allowing some cases that are currently prohibited?